test/units/contrib/inventory/test_ec2_inventory.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+
+import pytest
+
+
+@pytest.fixture
+def ec2_inventory():
+    # contrib's dirstruct doesn't contain __init__.py files
+    # import ec2 inventory manually
+    inventory_dir = os.path.abspath(
+        os.path.join(
+            os.path.dirname(__file__), '..', '..', '..', '..', 'contrib',
+            'inventory'))
+    sys.path.append(inventory_dir)
+
+    try:
+        import ec2
+    except ImportError as ex:
+        pytest.fail(ex)
```

Comment:
Since only `ImportError` is being handled here, use this instead:

```python
try:
    ec2 = pytest.importorskip("ec2")
finally:
    # rest of code here
```

That way the tests will be properly skipped if the import isn't available.

Reply:
fixed it, so the policy is like to skip test if `ImportError` happens, and do not fail it?

Reply:
Correct.

Reply:
Hi, is there anything improvement I can do for this PR?